### PR TITLE
docs: `integrations/retrievers` cleanup

### PR DESCRIPTION
**Pull request by leo-gan**
fixed several notebooks:
- headers
- formats

### CODE REVIEW

File: docs/extras/integrations/providers/scann.mdx
Comment (baskaryan) [inline docs/extras/integrations/providers/scann.mdx:1]: im not sure this is mentioned into e Google page is it?

File: docs/extras/integrations/providers/aws_dynamodb.mdx
Comment (baskaryan) [inline docs/extras/integrations/providers/aws_dynamodb.mdx:1]: can we add redirects for deleted pages to the relevant platform page

**Review by baskaryan — COMMENTED** [review]
thanks @leo-gan! few small comments

File: docs/extras/integrations/providers/scann.mdx
Comment (leo-gan) [reply docs/extras/integrations/providers/scann.mdx:1]: Oops, sorry. I've messed up with git and combined the #11312 PR in here. I'll split them again. Sorry for that.

File: docs/extras/integrations/providers/aws_dynamodb.mdx
Comment (leo-gan) [reply docs/extras/integrations/providers/aws_dynamodb.mdx:1]: Sure. I'll do this in the ##11312

Comment (leo-gan) [reply docs/extras/integrations/providers/aws_dynamodb.mdx:1]: unfortunately, the #11312 was merged. So, I place the page redirections in this PR.

File: docs/extras/integrations/providers/scann.mdx
Comment (leo-gan) [reply docs/extras/integrations/providers/scann.mdx:1]: unfortunately, the !11312 was merged.

**Review by efriis — APPROVED** [review]

**Pull request merged**